### DISCUSSION TITLE
Update File with Alert Actions

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#7345] [Unsupported Versions] [answered]
Hi,
I tried understanding the complicated alert actions from the Security Onion documentation but with no success.

I want to make an action that adds certain parameters from the alert to a file.

Does anyone know how that action would look like?

**Comment by dougburks** [comment, marked as answer]
You would need to build a web service that accepts parameters and writes them to a file. Once you have that web service in place, then you would follow the directions at https://docs.securityonion.net/en/2.3/soc-customization.html#action-menu to create an action that would send to your web service.
